### PR TITLE
KEDA scaling using SAS

### PR DESCRIPTION
**Pull request by nickdala**

### CODE REVIEW

File: infra/shared/terraform/modules/postresql/main.tf [hunk 1]
```diff
@@ -63,11 +63,9 @@ resource "azurerm_postgresql_flexible_server" "postgresql_database" {
 
     content {
       mode = "ZoneRedundant"
-      standby_availability_zone = 2
```
Comment (mnriem) [inline infra/shared/terraform/modules/postresql/main.tf:66]: What does this have to do with the KEDA scaling rule?

**Review by mnriem — APPROVED** [review]

File: infra/shared/terraform/modules/postresql/main.tf (hunk 1 above)
Comment (nickdala) [reply infra/shared/terraform/modules/postresql/main.tf:66]: This is a general improvement to the Terraform.
